### ISSUE TITLE
Increase the amount of medicine hospitals spawn

### ISSUE THREAD

**Issue by ISuckM8** [CleverRaven/Cataclysm-DDA#54108]
### Is your feature request related to a problem? Please describe.

Hospitals spawn less medicine than a pharmacy, real-life hospitals usually have at least one medicine room with a lot of medicine.

### Solution you would like.

Increase the amount of medicine hospitals have.

### Describe alternatives you have considered.

None.

### Additional context

![D1061_235_057_1200](https://user-images.githubusercontent.com/87550905/148439830-6a3601d0-dbaf-435d-978a-e5b2d1e1968f.jpg)

**Comment by Venera3** [comment]
With riots and societal disruption them being uderstocked on the high-throughput stuff isn't unreasonable - niche things like antiparasitics might be safe, but e.g. anesthetic  and other trauma supplies can run low surprisingly quickly  even in less-apocalyptic times.

**Comment by leemuar** [comment]
That's reasonable. But on the other hand - hospitals are kinda dangerous, with lots of Zed's inside. Clearing it out should probably give the player some good medical loot

> With riots and societal disruption them being uderstocked on the high-throughput stuff isn't unreasonable - niche things like antiparasitics might be safe, but e.g. anesthetic and other trauma supplies can run low surprisingly quickly even in less-apocalyptic times.

**Comment by ISuckM8** [comment]
> With riots and societal disruption them being uderstocked on the high-throughput stuff isn't unreasonable - niche things like antiparasitics might be safe, but e.g. anesthetic and other trauma supplies can run low surprisingly quickly even in less-apocalyptic times.

They still have less things than pharmacies, even things that wouldn't have been used too much like antiseptic and antiparasitics.

**Comment by Venera3** [comment]
Yeah, antiseptic is such a niche thing you don't need for literally any wound more complicated than an abrasion. I'm sure they were just swimming in the stuff at the end.
